common/src/main/java/io/github/foundationgames/automobility/controller/ControllerCompat.java

```diff
@@ -0,0 +1,40 @@
+package io.github.foundationgames.automobility.controller;
+
+public interface ControllerCompat {
```

Comment:
I'd call this AutomobileController or something without "compat" in the name, see comment further down for why